modules/math/src/geometry/attributes/compute-vertex-normals.ts

```diff
@@ -1,14 +1,17 @@
-/** @typedef {import('./compute-vertex-normals')} types */
 import {Vector3} from '@math.gl/core';
 import {GL} from '../constants';
 import {assert} from '../utils/assert';
 import {makePrimitiveIterator} from '../iterators/primitive-iterator';
 import {getPrimitiveModeType} from '../primitives/modes';
 import {getPositions} from './get-attribute-from-geometry';
 
-/** @type {types['computeVertexNormals']} */
+/**
+ * Computes vertex normals for a geometry
+ * @param param0
+ * @returns
+ */
 // eslint-disable-next-line max-statements
-export function computeVertexNormals({mode, indices, attributes}) {
+export function computeVertexNormals(mode: any, indices?: number, attributes?: {}): Float32Array {
```

Comment:
I think it can be breaking change. `computeVertexNormals` used to consume only one param as object, but now we should pass 3 arguments there.